example/example-edv-wallet.js

```diff
@@ -13,7 +13,7 @@ import MockHmac from '../tests/mock/hmac';
 
 import useStorageWallet from './use-storage-wallet';
 
-import { X25519KeyAgreementKey2019 } from '@digitalbazaar/x25519-key-agreement-key-2019';
+import { X25519KeyAgreementKey2020 } from '@digitalbazaar/x25519-key-agreement-key-2020';
```

Comment:
any danger that switching this up will break things?

Reply:
EDV server supports both 2019 and 2020 pairs, but in these examples its better to just use 2020. you can run tests in `master` atm with the latest EDV and its fine.

updated here because the EDV client library requires 2020 keys now